tests/rb/HttpInterface/api-general-graph-spec.rb

```diff
@@ -739,6 +741,38 @@ def delete_edge (waitForSync, graph_name, collection, key, options = {})
             doc.parsed_response['new']['value'].should eq("foo")
           end
 
+          it "can not create edge with unknown from vertex collection, waitForSync = #{sync}" do
+            v1 = create_vertex( sync, graph_name, user_collection, {})
+            v1 = v1.parsed_response['vertex']['_id']
+            doc = create_edge( sync, graph_name, friend_collection, "MISSING/v2", v1, {})
+            doc.parsed_response['error'].should eq(true)
+            doc.parsed_response['code'].should eq(400)
+            doc.parsed_response['errorMessage'].should include("_from vertex collection does not exist")
+            doc.parsed_response['errorNum'].should eq(1947)
+          end
+
+          it "can not create edge with unknown to vertex collection, waitForSync = #{sync}" do
+            v1 = create_vertex( sync, graph_name, user_collection, {})
+            v1 = v1.parsed_response['vertex']['_id']
+            doc = create_edge( sync, graph_name, friend_collection, v1, "MISSING/v2", {})
+            doc.parsed_response['error'].should eq(true)
+            doc.parsed_response['code'].should eq(400)
+            doc.parsed_response['errorMessage'].should include("_to vertex collection does not exist")
+            doc.parsed_response['errorNum'].should eq(1948)
+          end
+
+          it "replace edge (collection does not exist) not found, waitForSync = #{sync}" do
```

Comment:
replace edge [...] not found?

Reply:
Yes, will fix (test just got moved and expanded, but anyway will do :) )